public/app/features/dashboard-scene/serialization/angularMigration.ts

```diff
@@ -12,6 +12,17 @@ export function getAngularPanelMigrationHandler(oldModel: PanelModel) {
 
     if (!oldModel.options || Object.keys(oldModel.options).length === 0) {
       defaults(panel, oldModel.getOptionsToRemember());
+
+      // Some plugins rely on being able to access targets to set up the fieldConfig when migrating from angular.
+      const targetClone = cloneDeep(oldModel.targets);
+      Object.defineProperty(panel, 'targets', {
+        get: function () {
+          console.warn(
+            'Accessing the targets property when migrating a panel plugin is deprecated. Changes to this property will be ignored.'
+          );
+          return targetClone;
+        },
+      });
```

Comment:
this won't solve plugins wanting to access this in a migration handler? Only plugins that want to use targets in the render path, do we have such an example?